src/__tests__/applicationForms.int.test.ts

```diff
@@ -48,55 +55,32 @@ describe('/applicationForms', () => {
           ( 1, 2, '2022-08-20 12:00:00+0000' ),
           ( 2, 1, '2022-09-20 12:00:00+0000' )
       `);
-			await agent
+			const response = await agent
 				.get('/applicationForms')
 				.set(authHeader)
-				.expect(200, [
+				.expect(200);
+			expect(response.body).toMatchObject({
+				entries: [
 					{
-						createdAt: '2022-07-20T12:00:00.000Z',
+						createdAt: expectTimestamp,
```

Comment:
I think the test is stronger when it verifies that the exact data comes back associated with the right row. In other words, I prefer the literal. But that should not hold up merging this.

Reply:
I hear you -- the big issue is that the the json aggregation functions in psql appear to do timestamp-to-string conversion slightly differently from before; in particular they're returning things as +0:00 or +4:00 instead of `Z` -- which means the exact string representing the time might vary from machine to machine.

Reply:
That might be because the session time zone varies depending on the client machine. You can `SET TIME ZONE 'UTC';` on a per-session basis; see https://stackoverflow.com/a/27351341 . I haven't looked into how to do this outside of `psql(1)`, though, and in particular I haven't looked into how to do so with the library used here.